packages/web/src/pages/register.js

```diff
@@ -0,0 +1,104 @@
+// Module imports
+import { useCallback } from 'react'
+
+
+
+
+
+// Local imports
+import { Field } from 'components/Field'
+import { Form } from 'components/Form'
+import { FormButton } from 'components/FormButton'
+import { useAuth } from 'contexts/AuthContext'
+import { useRedirectOnLoggedIn } from 'hooks/useRedirectOnLoggedIn'
+
+
+
+
+
+export default function RegisterPage() {
+	const {
+		isRegistering,
+		register,
+		validateEmail,
+		validateUsername,
+	} = useAuth()
+
+	useRedirectOnLoggedIn()
+
+	const handleSubmit = useCallback(formState => {
+		const {
+			isValid,
+			values,
+		} = formState
+
+		if (isValid) {
+			register(values)
+		}
+	}, [register])
+
+	const handleValidateEmail = useCallback(async value => {
```

Comment:
handleValidateEmail is unused

Reply:
Yup, I've got it fixed in my next PR. Good call out, because this was a silly oversight. 🤦🏻